apps/api/src/app/workflows-v2/usecases/delete-workflow/delete-workflow.command.ts

```diff
@@ -4,5 +4,5 @@ import { IsDefined, IsString } from 'class-validator';
 export class DeleteWorkflowCommand extends EnvironmentWithUserObjectCommand {
   @IsString()
   @IsDefined()
-  workflowId: string;
+  workflowIdOrIdentifier: string;
```

Comment:
```suggestion
  workflowIdOrSlug: string;
```

Reply:
As discussed offline we will proceed with `IdentifierOrInternalId`

Reply:
done